src/main/resources/feedbackQuestionConstSumSubmissionFormOptionFragment.html

```diff
@@ -1,7 +1,7 @@
-<div class="form-group">
-    <label class="col-sm-4 col-md-2 control-label" ${constSumOptionVisibility}>${constSumOptionValue}</label>
-    <div class="col-sm-6 col-md-3">
-        <input type="number" class="form-control pointsBox" ${disabled} 
+<div class="margin-bottom-15px flex">
```

Comment:
form-group `.form-group {
    margin-left: -15px;
    margin-right: -15px;
   margin-bottom: 15px;
}` is removing the paddings caused by nested columns (col-sm-x). As I am not using nested columns here, it is not required here. But we need `margin-bottom: 15px`, so we need it explicitly.